### PR TITLE
Submitting Grid Printer

### PR DESCRIPTION
**Pull request by djgordon-maker**

### CODE REVIEW

**Comment by djgordon-maker** [comment]
Submitting all Lesson 2 activites

File: students/daniel_gordon/lesson01/break_me.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+def breakName():
```
Comment (charlesrobison) [inline students/daniel_gordon/lesson01/break_me.py:1]: Make sure all function names are snake_case in Python

File: students/daniel_gordon/lesson02/fizz_buzz.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+for i in range(1,101):
+    text = ''
+    if i % 3 == 0:
+        text += 'Fizz'
+    if i % 5 == 0:
+        text += 'Buzz'
+    if text == '':
+        print(i)
+    else:
+        print(text)
```
Comment (charlesrobison) [inline students/daniel_gordon/lesson02/fizz_buzz.py:10]: This works :)

File: students/daniel_gordon/lesson02/grid_painter.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
+def PrintGrid(cells, size):
```
Comment (charlesrobison) [inline students/daniel_gordon/lesson02/grid_painter.py:1]: Should be print_grid(cells, size):

Same for all function names below

File: students/daniel_gordon/lesson02/grid_painter.py [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    return True
+
+def PrintCell(cells, size):
+    for i in range(size):
+        for i in range(cells):
+            print('|', ' '*size, end = ' ')
+        print('|')
+    return True
+
+PrintGrid(10,1)
```
Comment (charlesrobison) [inline students/daniel_gordon/lesson02/grid_painter.py:21]: Looks like you skipped the first few steps in this exercise - but that's okay!

File: students/daniel_gordon/lesson02/series.py [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+    #test a generalized sequence
+    assert sum_series(0,4,2) == 4
+    assert sum_series(1,4,2) == 2
+    assert sum_series(2,4,2) == 6
+    assert sum_series(3,4,2) == 8
+    assert sum_series(4,4,2) == 14
+    assert sum_series(5,4,2) == 22
+    assert sum_series(6,4,2) == 36
+    
+    print("test successfull")
```
Comment (charlesrobison) [inline students/daniel_gordon/lesson02/series.py:55]: Nice!

**Pull request merged**